src/web/web-service/web_server.cpp

```diff
@@ -195,7 +199,7 @@ void WebServer::DefaultHttpResponse(void)
 
         } catch (const std::exception &e)
         {
-            std::string content = "Could not open path " + request->path + ": " + e.what();
+            std::string content = "Could not open path";
```

Comment:
how about include the escaped path?

Reply:
Escaping url turns out to be tricky and is not really that necessary